src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -565,11 +565,25 @@ void CodegenNeuronCppVisitor::print_namespace_stop() {
     printer->pop_block();
 }
 
+void CodegenNeuronCppVisitor::append_conc_write_statements(
+    std::vector<ShadowUseStatement>& statements,
+    const Ion& ion,
+    const std::string& /* concentration */) {
```

Comment:
still needs to be kept?

Reply:
What needs to be kept? The unused argument `concentration`? Yes, CoreNEURON generates complately different code and it needs it. At least IIRC.

Reply:
yes, that's what I was referring to! my bad, I missed it's neuron-specific visitor!